http/swagger.yml

```diff
@@ -6615,6 +6615,20 @@ components:
             $ref: "#/components/schemas/Cells"
         labels:
             $ref: "#/components/schemas/Labels"
+    CreateDashboardRequest:
```

Comment:
One possible change would be to have Something like a `DashboardPartial` or `DashboardBody` that both Create and Dashboard could reference so that we don't have to update fields in multiple locations whenever we add a field to dashboards.